src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1014,21 +1014,11 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
     @Test
     public void testClearLazyLoadCacheInDetailAST() throws Exception {
-        final DefaultConfiguration checkConfig1 =
-            createModuleConfig(CheckWhichDoesNotRequireCommentNodes.class);
-        final DefaultConfiguration checkConfig2 =
-            createModuleConfig(CheckWhichRequiresCommentNodes.class);
```

Comment:
Why did we lose one of these "check" modules?

Reply:
Sorry, must've dropped the latest commit during rebase. Now, added config for module